src/app/components/ColorSelector/ColorSelector.tsx

```diff
@@ -1,15 +1,22 @@
-import React, { ReactNode } from 'react';
+import React, { Dispatch, SetStateAction } from 'react';
+
+const colors = ['red', 'pink', 'grey', 'lime'];
 
 type ColorSelectorProps = {
-  children: ReactNode;
+  onSelectColor: Dispatch<SetStateAction<string>>;
```

Comment:
Interesting type. I would prefer a not so specific type like:
```suggestion
  onSelectColor: (color: string) => void;
```